aquarius/events/util.py

```diff
@@ -10,12 +10,12 @@
 from ocean_lib.config import Config
 from ocean_lib.config_provider import ConfigProvider
 from ocean_lib.web3_internal.contract_handler import ContractHandler
+import ocean_lib.web3_internal.utils
```

Comment:
why not use the same "from ... import ..." syntax?

Reply:
Name collision between `ocean_lib.web3_internal.utils.get_network_name` and `aquarius.events.util.get_network_name`

Reply:
you can use "as" I think